src/main/java/org/synyx/urlaubsverwaltung/dev/DemoDataProperties.java

```diff
@@ -10,6 +10,8 @@ public class DemoDataProperties {
      */
     private boolean create = true;
 
+    private boolean localDevelopment = true;
```

Comment:
think it's better to disable it by default and enable it when running app with demodata profile?

Reply:
fixed